doc/installation/install_options.rst

```diff
@@ -20,9 +20,10 @@ For more detailed information please see below: ``Python3 Binding (PyNEST)``
 Choice of compiler
 ------------------
 
-The default compiler for NEST is GNU gcc/g++, but NEST has also successfully
-been compiled with other compilers, including Intel icc/icpc, Pathscale,
-Portland and IBM compilers.
+The default compiler for NEST is GNU gcc/g++. Version 7 or higher is required
+due to the presence of bugs in earlier versions that prevent the compilation
+from succeeding. NEST has also successfully been compiled with other 
+compilers, including Intel icc/icpc, Pathscale, Portland and IBM compilers.
```

Comment:
We should update the list of "other compilers" mentioned here. First, we definitely should mention Clang, since we now explicitly test with it. Some of us probably use the Intel compiler occasionally, but I think it is a long time since anyone has tested Pathscale, Portland and probably also IBM, so I would take those out.